pandas/core/tools/datetimes.py

```diff
@@ -434,9 +449,11 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
                     yearfirst=yearfirst,
                     require_iso8601=require_iso8601
                 )
-
             if is_datetime64_dtype(result) and box:
                 result = DatetimeIndex(result, tz=tz, name=name)
+            # GH 6415
+            elif arg_is_series and utc:
```

Comment:
OK, but for for step 2, why do you then need to have `_convert_listlike` handle Series vs arrays/lists of strings differently? 
(As far as I can see, this is because for a Series the values are not converted to a DatetimeIndex at the end of `_convert_listlike`, and so the handling of utc has to be done afterwards again, but I can be wrong)